Libs/MRML/Core/Testing/vtkMRMLTransformNodeTest1.cxx

```diff
@@ -10,10 +10,43 @@
 
 =========================================================================auto=*/
 
+#include "vtkMRMLBSplineTransformNode.h"
 #include "vtkMRMLCoreTestingMacros.h"
+#include "vtkMRMLScene.h"
 #include "vtkMRMLTransformNode.h"
 
 #include <vtkGeneralTransform.h>
+#include <vtkMatrix4x4.h>
+#include <vtkNew.h>
+#include <vtkTransform.h>
+
+bool Matrix4x4AreEqual(vtkMatrix4x4 *m1, vtkMatrix4x4 *m2)
```

Comment:
We should probably collect all the utilities we find useful and put them in
vtkAddon with the idea that they eventually make their way to vtk proper.
Fuzzy matrix comparison is definitely one of the candidates, but I'm sure
there are others.

On Mon, Dec 7, 2015 at 2:07 PM, Andras Lasso <notifications@github.com>
wrote:

> In Libs/MRML/Core/Testing/vtkMRMLTransformNodeTest1.cxx
> <https://github.com/Slicer/Slicer/pull/430#discussion_r46861653>:
>
> >  #include "vtkMRMLTransformNode.h"
> >
> >  #include <vtkGeneralTransform.h>
> > +#include <vtkMatrix4x4.h>
> > +#include <vtkNew.h>
> > +#include <vtkTransform.h>
> > +
> > +bool Matrix4x4AreEqual(vtkMatrix4x4 *m1, vtkMatrix4x4 *m2)
>
> That one does not have a tolerance in the comparison, so I'm not sure how
> well it would work. It would make sense to add some tolerance in
> vtkMRMLSliceNode::Matrix4x4AreEqual(), too, but I would hesitate to do that
> in this commit.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Slicer/Slicer/pull/430/files#r46861653>.
>